src/Application.vala

```diff
@@ -104,19 +104,21 @@ public class Terminal.Application : Gtk.Application {
                         if (!terminal.is_init_complete ()) {
                             terminal.set_init_complete ();
                         } else {
+                            var process_string = _("Process exited successfully");
```

Comment:
I know this is technically correct, but I kind of hate the "succeeded successfully mentality; is it accurate enough to just say `Process completed` since you'd think we'd be more explicit about completing with errors?

```suggestion
                            var process_string = _("Process completed");
```